src/errors.ts

```diff
@@ -15,10 +18,141 @@ export async function errors(
   logger.error(error.message);
 
   if (process.env.NODE_ENV === 'production') {
-    return res.status(500).send();
+    return res.status(StatusCodes.INTERNAL_SERVER_ERROR).send();
   } else {
-    return res.status(500).send({
+    return res.status(StatusCodes.INTERNAL_SERVER_ERROR).send({
       message: error.message,
     });
   }
 }
+
+// TODO Errors must be put in a shared package that the SDK also uses
+export type ErrorPayload = {
+  error: string;
+  status: StatusCodes;
+  message: string;
+};
+
+export const REQUEST_VALIDATION_ERROR: ErrorPayload = {
+  status: StatusCodes.BAD_REQUEST,
+  error: 'request-validation-error',
+  message: 'The request payload is incorrect',
+};
+
+const asErrors = <T>(et: {
+  [K in keyof T]: Pick<ErrorPayload, 'status' | 'message'>;
+}) => et;
+
+const ERRORS = asErrors({
```

Comment:
How about we use constants to set the keys?

Like:

```
export const ROUTE_NOT_FOUND = 'route-not-found';
```

Then we can use `ROUTE_NOT_FOUND` both in the key here and wherever it's used using the `sendError` function?

Reply:
nvm, we have types.